Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -884,10 +884,9 @@ def get_mapping_fields_command():
     return mapping_response
 
 
-def get_modified_remote_data_command(client, args, xdr_delay: int = 1):
+def get_modified_remote_data_command(client, args, mirroring_last_run: Dict[str, Any] = {}, xdr_delay: int = 1):
     remote_args = GetModifiedRemoteDataArgs(args)
-    integration_context = demisto.getIntegrationContext()
-    last_update: str = integration_context.get('mirroring_last_update', remote_args.last_update)
+    last_update: str = mirroring_last_run.get('mirroring_last_update', remote_args.last_update)
```

Comment:
why setting the mirroring_last_run to be a dict? its a single value, why not just the value itself under the last run with the "mirroring" key?